src/main/java/com/hubspot/jinjava/lib/filter/PrettyPrintFilter.java

```diff
@@ -48,7 +51,17 @@ public Object filter(Object var, JinjavaInterpreter interpreter, String... args)
     ) {
       varStr = Objects.toString(var);
     } else {
-      varStr = new POJONode(var).toPrettyString();
+      try {
+        varStr =
+          interpreter
+            .getConfig()
+            .getObjectMapper()
+            .registerModule(new JavaTimeModule())
```

Comment:
Does this mean we register the module every time the function is called? Is this an expensive operation?